.circleci/continue_config.yml

```diff
@@ -331,8 +331,8 @@ workflows:
       - win_e2e_tests:
           matrix:
             parameters:
-              #3.10 fails with win-build, hence reverting it to 3.10
-              python_version: ['3.7', '3.8', '3.9', '3.10']
+              #removing 3.7 for now since it's failing circleCI and we are planning to soon stop support for 3.7
```

Comment:
@merelcht  -- they were WindowsPath failures related to subprocess and stuff.
 I also realised they were only failing for lower-bound requirements. and that's like super niche and not at all important thing to spend time on. 
however I agree we shouldn't remove win-37 completely because of it so thanks for the nudge.  what I have done now in the code is if you are on windows and using python 3.7, then that scenario which checks with lower-bounds just gets skipped.

Reply:
Thanks for explaining! I think that's a good solution 👍